docs/general/session.md

```diff
@@ -35,4 +35,36 @@ backends can link the two sessions.
 <!-- END AUTOGENERATED TEXT -->
 <!-- endsemconv -->
 
+## Session Events
+
+### Session Start Event
+
+![Experimental](https://img.shields.io/badge/-experimental-blue)
+
+`event.name`=`session.start`
+
+For instrumentation that tracks users behavior during user sessions, a `session.start` event MUST be emitted
+every time a session is created. When a new session is created as a continuation of a prior session,
+the `session.previous_id` SHOULD be included in the event. The values of `session.id` and `session.previous_id`
+MUST be different.
+
+| Body field                                                    | Type   | Description                                          | [Requirement Level](https://opentelemetry.io/docs/specs/semconv/general/attribute-requirement-level/) |
+|---------------------------------------------------------------|--------|------------------------------------------------------|-------------------------------------------------------------------------------------------------------|
+| [`session.id`](/docs/attributes-registry/session.md)          | string | The ID of the new session being started.             | `Required`                                                                                            |
```

Comment:
Having this in the body field, will this preclude it from also being an attribute?

Reply:
It's a good question. As written now, it's intended to be in the body because I think that's what people were suggesting in the other PR, and we have the body. It wouldn't make sense to me to have it in both places .

Reply:
The way this is being used, it definitely belongs in the body given it has a specific meaning in the context of this event, which is why we might want to use a different name to disambiguate it from a `session.id` being in the attributes, which signals the session the event was fired in.